src/mono/wasm/testassets/BlazorHostedApp/BlazorHosted.Server/BlazorHosted.Server.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <NoWarn>CS8633</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="9.0.0-alpha.1.23614.6" />
+    <PackageReference Include="Serilog.Extensions.Logging.File" Version="3.0.0" />
```

Comment:
Do we need serilog?

Reply:
Yes, we're writing logs to file on the server side, in case there was a problem in the server -> client communication and client would not be able to collect the info.